scene/2d/camera_2d.cpp

```diff
@@ -363,15 +363,15 @@ Camera2D::AnchorMode Camera2D::get_anchor_mode() const {
 	return anchor_mode;
 }
 
-void Camera2D::set_rotating(bool p_rotating) {
-	rotating = p_rotating;
+void Camera2D::set_ignore_rotation(bool p_ignore) {
+	ignore_rotation = p_ignore;
 	Point2 old_smoothed_camera_pos = smoothed_camera_pos;
 	_update_scroll();
 	smoothed_camera_pos = old_smoothed_camera_pos;
 }
 
-bool Camera2D::is_rotating() const {
-	return rotating;
+bool Camera2D::is_rotation_ignored() const {
+	return ignore_rotation;
```

Comment:
Why both `rotation_ignored` and `ignore_rotation` name orders?

Reply:
I'm not sure either, mostly because all other booleans do not begin with "enable", but rotation_ignored sounds wrong at the same time.

I will rename the getter as suggested, however.